protocols/118e8c/118e8c.ot2.apiv2.py

```diff
@@ -70,8 +77,8 @@ def run(ctx):
     # tips and p300 multi
     tips300 = [ctx.load_labware('opentrons_96_filtertiprack_200ul', '11')]
     tips20 = [ctx.load_labware('opentrons_96_filtertiprack_20ul', '10')]
-    p300s = ctx.load_instrument('p300_single_gen2', 'right', tip_racks=tips300)
-    p20m = ctx.load_instrument('p20_multi_gen2', 'left', tip_racks=tips20)
+    p300s = ctx.load_instrument('p300_single_gen2', 'left', tip_racks=tips300)
```

Comment:
Not critical but have you considered making the mount positions their own parameters?